### PR TITLE
[zh] Translate docker to crictl map page

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

**Comment by tengqm** [comment]
Preview: https://deploy-preview-31943--kubernetes-io-main-staging.netlify.app/zh/docs/reference/tools/map-crictl-dockercli/

File: content/zh/docs/reference/tools/map-crictl-dockercli.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+此页面已被废弃，将在 Kubernetes 1.27 版本删除。
+{{</note>}}
+
+<!--
+`crictl` is a command-line interface for {{<glossary_tooltip term_id="cri" text="CRI">}}-compatible container runtimes.
+You can use it to inspect and debug container runtimes and applications on a
+Kubernetes node. `crictl` and its source are hosted in the
+[cri-tools](https://github.com/kubernetes-sigs/cri-tools) repository.
+-->
+`crictl` 是 {{<glossary_tooltip term_id="cri" text="CRI">}}-兼容的容器运行时的一种命令行接口。
```
Comment (howieyuen) [inline content/zh/docs/reference/tools/map-crictl-dockercli.md:26]:
```suggestion
`crictl` 是一种兼容 {{<glossary_tooltip term_id="cri" text="CRI">} 的容器运行时的命令行接口。
```

**Review by howieyuen — CHANGES_REQUESTED** [review]

**Pull request merged**